default.pp

```diff
@@ -0,0 +1,4 @@
+
+class { 'java':} ->
```

Comment:
that file looks totally obsolete (you have to know that this is executed by the vagrantfile)